core/src/main/resources/lib/form/file.jelly

```diff
@@ -23,29 +23,32 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form"
+         xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary">
   <st:documentation>
     Generates an input field <code>&lt;input type="file" ... /></code>
+    All unknown attributes are passed through to the field.
 
     <st:attribute name="field">
       Used for databinding.
     </st:attribute>
-    <st:attribute name="jsonAware">
-      Enable structured form submission.
+    <st:attribute name="clazz">
+      Additional CSS class(es) to add.
     </st:attribute>
-    <st:attribute name="accept">
```

Comment:
Annoyingly the IntelliJ plugin doesn't support the morph tag library and marks anything using accept as an error 😢.

<img width="345" alt="image" src="https://user-images.githubusercontent.com/21194782/210435615-e06e5e6e-5ab0-4818-9531-85d5ed72160d.png">